### ISSUE TITLE
Review of the requirements and related diagrams

### ISSUE THREAD

**Issue by serkan-ozel-2015400123** [bounswe/bounswe2018group9#139]
As you know, couple of days ago we have finished writing the report. Prior to that I have revised the requirements a little bit. What I did was only removing the redundant requirements and detailing some of the existing ones. The detailing part is not important to add new diagrams sequence diagrams.. for it. Therefore, if we change our requirements now, all we need is to update the references to the old requirements to make it refer to new one. 

Can we find out where we would need to revise if we change the requirements to the new one? Also, can we distribute this task among us? 

Or should we do this after the final, in case you are familiar with the old requirements and expect questions from the old one?

**Comment by akaogluozgur** [comment]
During the initial draft of object diagram, I have spend considerable amount of time to go over requirements and produced the diagram. At that time this issue happened again so I had to revise all requirements from begining. Now , Why did you change enumeration of requirements at the first place, when you change a content that is already in our wiki, you should be the one that consider backward compatibility. Maintaining requirements is easier than updating rest of our wikipage.

**Comment by gumityolcu** [comment]
I thought we had submitted the report with the former version of the requirements because we reached the consensus that changing the requirements had a great overhead. Since we submitted the report, I think we should study the original requirements for the final exam. I propose -and we probably will have a homework forcing us to- revising the requirements and creating the last version next year, just before we start the project, which will be good practice because we will have the chance to remember clearly and precisely the limits of what we want to produce.
